docs/source/index.rst

```diff
@@ -90,6 +90,7 @@ Read more about this in :doc:`advanced/jit`.
    notebooks/Logistic_Regression
    notebooks/Custom_Networks
    examples
+   tutorials
```

Comment:
`notebooks/*` sections are also called tutorials, so after adding this line table of contents will look like following:

Getting Started

* Installation and Setup
* Objax Basics Tutorial
* Logistic Regression
* Creating Custom Networks for Multi-Class Classification
* Code Examples
* Tutorials

...

-------------------

I think maybe this **tutorials** item should be named `Additional tutorials` to distinguish from basic tutorials which we wanted user to read first.

Reply:
Changed the title of tutorial.rst to "Additional tutorials"